.changelog/12933.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ci: use correct variable syntax for build-distros job
```

Comment:
I don't think we need a changelog entry for this, as it's not a user-facing change?

Reply:
I thought we had included `ci` changes previously, but it not being user-facing makes more sense.